README.md

```diff
@@ -8,7 +8,11 @@
 
 ## Getting Started
 
-To build it, we use [mill](http://www.lihaoyi.com/mill/). I you want to run the specification tests, just type:
+To build it, we use [mill](http://www.lihaoyi.com/mill/). Mill is not available on on Unix boxes, but you can you use curl to install it (see Installation instructions in the [doc](http://www.lihaoyi.com/mill/)):
+
+    sh -c '(echo "#!/usr/bin/env sh" && curl -L https://github.com/lihaoyi/mill/releases/download/0.3.6/0.3.6) > $HOME/bin/mill && chmod +x $HOME/bin/mill'
```

Comment:
I will have to think about upgrading this line when I upgrade mill :(

Reply:
agree. Unfortunately, github does not support `https://github.com/lihaoyi/mill/releases/download/latest/latest`

Reply:
I actually doubt this makes sense here, because, it is simply copy/paste of what is in the mill documentation.

Reply:
I like self-contained getting started without opening any external URL.

Do you want me to remove this?

Reply:
Well, it is not really, as OSX, Arch Linux, and FreeBSD users don't have the preferred way of doing it here, so this looks like this is the only way to isntall mill (which it is not). I would remove it. Your readme can never be self contained, unless it integrates the documentation for every used tool. And I don't want to do that.

Reply:
no need to further discuss. removed.